src/views/SemanticWorkflow/Implementations/InputsTab.lua

```diff
@@ -0,0 +1,483 @@
+---@type InputsTab
+---@diagnostic disable-next-line: assign-type-mismatch
+local __impl = __impl
+
+__impl.name = "Inputs"
+__impl.help_key = "INPUTS_TAB"
+
+---@type FrameListGui
+local FrameListGui = dofile(views_path .. "SemanticWorkflow/Definitions/FrameListGui.lua")
+local Section = dofile(views_path .. "SemanticWorkflow/Definitions/Section.lua")
+
+---constants---
```

Comment:
If you **must** designate regions, do this:

```lua
--#region Constants
--#endregion
```

Some of the regions here just aren't necessary, since they only encompass one function.